parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -353,6 +364,45 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(4)]
+		#[pallet::weight(T::WeightInfo::import_execution_header())]
+		#[transactional]
+		pub fn import_execution_header_capella(
```

Comment:
I think it would be cleaner to support only one `import_execution_header()` extrinsic, instead of one for each version.

Then we can pass the following header type to `import_execution_header(origin, header)`:

```
enum VersionedHeaderUpdate {
    Capella(HeaderUpdateOfCapella)
}
```

And then the extrinsic can inspect `VersionedHeaderUpdate` to know which version needs to be processed.